### ISSUE TITLE
About - Mission Page Content

### ISSUE THREAD

**Issue by brianking** [OSLC/oslc-site-hugo#18]
Home -> About -> Mission

Based on:
https://open-services.net/wiki/steering-committee/Vision/

Open question: Are mission and vision the same thing?

**Comment by brianking** [comment]
Waiting on issue #114 before doing this and the other About sub-pages.

**Comment by brianking** [comment]
@berezovskyi What do you think we should do here? See first comment here, and #111.

**Comment by jamsden** [comment]
Vision is what you are trying to achieve, mission is how you plan to achieve it - what vs. how. These can be simple, broad summary statements about the open-services.net community.

http://open-services.net/wiki/steering-committee/Vision/ is a longer term future vision of the OASIS OSLC Member Section Steering Committee. This expresses the StC vision as a contribution to the vision for the open-services.net community.

So these are different things, the community vs. the StC visions.

**Comment by axelreichwein** [comment]
The About section should be focused on the people in the OSLC community and its vision. The current "Why OSLC" and "FAQ" subsections of the About section should become top-level menu items similar to About, etc...

**Comment by jamsden** [comment]
Doesn't need to be a live blockers

**Comment by gabrielfdac** [comment]
This is already done, see http://oslc.co/about/#mission
Closing
